vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -194,85 +226,92 @@ func (d *VolumeDriver) UnmountVolume(name string) error {
 // (until Mount is called).
 // Name and driver specific options passed through to the ESX host
 
-// Create - create a volume.
+// Create submits a volume creation request.
 func (d *VolumeDriver) Create(r volume.Request) volume.Response {
 
 	if r.Options == nil {
 		r.Options = make(map[string]string)
 	}
+
 	// If cloning a existent volume, create and return
-	if _, result := r.Options["clone-from"]; result == true {
-		errClone := d.ops.Create(r.Name, r.Options)
-		if errClone != nil {
-			log.WithFields(log.Fields{"name": r.Name, "error": errClone}).Error("Clone volume failed ")
-			return volume.Response{Err: errClone.Error()}
+	if _, ok := r.Options["clone-from"]; ok {
+		if err := d.ops.Create(r.Name, r.Options); err != nil {
+			log.WithField("name", r.Name).WithError(err).Error(
+				"Clone volume failed")
+			return volume.Response{Err: err.Error()}
 		}
 		return volume.Response{Err: ""}
 	}
 
 	// Use default fstype if not specified
-	if _, result := r.Options["fstype"]; result == false {
+	if _, ok := r.Options["fstype"]; !ok {
 		r.Options["fstype"] = fs.FstypeDefault
 	}
 
 	// Get existent filesystem tools
 	supportedFs := fs.MkfsLookup()
 
 	// Verify the existence of fstype mkfs
-	mkfscmd, result := supportedFs[r.Options["fstype"]]
-	if result == false {
-		msg := "Not found mkfs for " + r.Options["fstype"]
-		msg += "\nSupported filesystems found: "
-		validfs := ""
+	mkfscmd, ok := supportedFs[r.Options["fstype"]]
+	if !ok {
+		buf := &bytes.Buffer{}
+		fmt.Fprintf(buf, "Not found mkfs for %s\n", r.Options["fstype"])
+		fmt.Fprint(buf, "Supported filesystems found: ")
+		lenSupportedFS := len(supportedFs)
+		x := 0
 		for fs := range supportedFs {
-			if validfs != "" {
-				validfs += ", " + fs
-			} else {
-				validfs += fs
+			fmt.Fprint(buf, fs)
+			if x < lenSupportedFS-1 {
+				fmt.Fprint(buf, ", ")
 			}
+			x++
 		}
-		log.WithFields(log.Fields{"name": r.Name,
-			"fstype": r.Options["fstype"]}).Error("Not found ")
-		return volume.Response{Err: msg + validfs}
+		log.WithFields(log.Fields{
+			"name":   r.Name,
+			"fstype": r.Options["fstype"],
+		}).Error("Not found")
+		return volume.Response{Err: buf.String()}
 	}
 
-	errCreate := d.ops.Create(r.Name, r.Options)
-	if errCreate != nil {
-		log.WithFields(log.Fields{"name": r.Name, "error": errCreate}).Error("Create volume failed ")
-		return volume.Response{Err: errCreate.Error()}
+	if err := d.ops.Create(r.Name, r.Options); err != nil {
+		log.WithField("name", r.Name).WithError(err).Error(
+			"Create volume failed")
```

Comment:
Hi @govint,

Which white spaces? Any white space you see there is due to [`gofmt`](https://golang.org/cmd/gofmt/) and should remain there if your team and this project adheres to it. I have not yet encountered a Go project that does not since `gofmt` is the official syntax tool for Go.